src/devhub/entity/addon/kanban/post_board.jsx

```diff
@@ -9,75 +13,265 @@ const postTagsToIdSet = (tags) => {
+                          isTableView: true,
+                          columnId,
+                          data: postData,
+                        }}
+                        key={postData.id}
+                      />
+                    ))}
+                </tbody>
+              </table>
+            </div>
+          )}
+        </div>
+      );
+    } else {
+      return (
+        <div
+          className="col-3"
+          style={{ minWidth: "300px" }}
+          key={`column-${columnId}-view`}
+        >
+          <div className="card rounded-4">
+            <div
+              className={[
+                "card-body d-flex flex-column gap-3 p-2",
+                "border border-1 border-secondary rounded-4",
+              ].join(" ")}
+              style={{ height: "75vh" }}
+            >
+              <span className="d-flex flex-column py-1">
+                <h6 className="card-title h6 d-flex align-items-center gap-2 m-0">
+                  {column.title}
 
-                <span className="badge rounded-pill bg-secondary">
-                  {column.postIds.length}
-                </span>
-              </h6>
-
-              <p class="text-secondary m-0">{column.description}</p>
-            </span>
-
-            <div class="d-flex flex-column gap-2">
-              {column.postIds?.map((postId) => (
-                <Widget
-                  src={`${REPL_DEVHUB}/widget/devhub.entity.addon.${metadata.ticket.type}`}
-                  props={{ metadata: { id: postId, ...metadata.ticket } }}
-                  key={postId}
-                />
-              ))}
+                  <span className="badge rounded-pill bg-secondary">
+                    {column.postIds.length}
+                  </span>
+                </h6>
+
+                <p class="text-secondary m-0">{column.description}</p>
+              </span>
+
+              <div
+                class="d-flex flex-column gap-2"
+                style={{ overflow: "scroll" }}
+              >
+                {data.length === column.postIds.length &&
+                  data.map((postData) => (
+                    <Widget
+                      src={`${REPL_DEVHUB}/widget/devhub.entity.addon.${metadata.ticket.type}`}
+                      props={{
+                        metadata: { id: postData.postId, ...metadata.ticket },
+                        isTableView: false,
+                        data: postData,
+                      }}
+                      key={postData.postId}
+                    />
+                  ))}
+              </div>
             </div>
           </div>
         </div>
-      </div>
-    )
-  );
+      );
+    }
+  });
 
   return (
     <div>
-      <div className="d-flex flex-column align-items-center gap-2 py-4">
-        <h5 className="h5 d-inline-flex gap-2 m-0">
+      <div className="d-flex flex-column align-items-center gap-2 pb-4 w-100">
+        {/* <div class="form-check">
```

Comment:
We can remove the table view switch completely, since it will go in a different addon

Reply:
I would like to keep it in comments, may be after the proposals and trustees dashboard flow, we revisit it and improve it

Reply:
It's not good practice to clutter the main branch with commented code. Instead you can use version control to still "keep" the code. I would remove it for now. Than later when you want to revisit it you can run `git revert <remove commit id>`. This way you still have access to the code but the main branch will be clean.

Reply:
done